lib/rust/parser/debug/tests/parse.rs

```diff
@@ -19,31 +19,29 @@
 
 mod metadata;
 
-use enso_parser_debug::to_s_expr;
-use lexpr::sexp;
+use enso_parser_debug::test::block;
+use enso_parser_debug::test::expect_invalid_node;
+use enso_parser_debug::test::expect_multiple_operator_error;
+use enso_parser_debug::test::expect_valid;
+use enso_parser_debug::test::module;
+use enso_parser_debug::test::parse;
+use insta::assert_snapshot;
 
 
 
 // ===========================
 // === Test support macros ===
 // ===========================
 
-/// Parses input as a sequence of S-expressions, and wraps it in a `BodyBlock`.
-macro_rules! block {
-    ( $($statements:tt)* ) => {
-        sexp![(BodyBlock #( $( $statements )* ) )]
-    }
-}
-
-macro_rules! test {
+macro_rules! test_module {
     ( $code:expr, $($statements:tt)* ) => {
-        test($code, block![$( $statements )*])
+        assert_snapshot!(module($code), $( $statements )*)
```

Comment:
I think the `@` sign should be here, not in usage place. Otherwise, someone looking at this could write 

```rs
fn nothing() {
    test_module!("", "(BodyBlock #(()))");
}
```
Which will make snapshot called "" and always comparing "(BodyBlock #(()))"

If not possible because `cargo insta review` stops working, then at least parameters should be renamed and solid docs should be added.